internal/internal_activity.go

```diff
@@ -240,9 +240,17 @@ func getValidatedActivityFunction(f interface{}, args []interface{}) (*ActivityT
 	return &ActivityType{Name: fnName}, input, nil
 }
 
+func getReflectTypeKind(fType reflect.Type) reflect.Kind {
```

Comment:
you can just name this "kind", might be simpler to read. We can infer it is for reflect.Type from the function argument.

Reply:
sure